docs/docs/configurations.md

```diff
@@ -0,0 +1,348 @@
+
+Parameters to pass to Pytorch lightning's [`EarlyStopping`](https://github.com/PyTorchLightning/pytorch-lightning/blob/c7451b3ccf742b0e8971332caf2e041ceabd9fe8/pytorch_lightning/callbacks/early_stopping.py#L35) if `early_stopping` is `True`. The default values are specified in the `EarlyStoppingConfig` <!-- TODO: add link to github source code><!--> class: `EarlyStoppingConfig(monitor='val_macro_f1', patience=3, verbose=True, mode='max')`
+
+#### `tensorboard_log_dir (str, optional)`
+
+Pytorch Lightning can log to a local file system in TensorBoard format with [TensorBoardLogger](https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.loggers.tensorboard.html). The directory in which to save these logs is set to `zamba/models/<tensorboard_log_dir>/`. Defaults to `tensorboard_logs`
+
+#### `weight_download_region [us|eu|asia]` 
+
+Because `zamba` needs to download pretrained weights for the neural network architecture, we make these weights available in different regions. `us` is the default, but if you are not in the US you should use either `eu` for the European Union or `asia` for Asia Pacific to make sure that these download as quickly as possible for you.
+
+#### `cache_dir (FilePath, optional)`
+
+The directory where the trained model will be saved. If it is `None` (the default), the model will be cached to an automatic temp directory at `~/.cache/zamba`
+
+#### `split_proportions (dict(str, int), optional)`
+
+The proportion of data to use during training, validation, and as a holdout set. Defaults to `{"train": 3, "val": 1, "holdout": 1}`
+
+#### `save_directory (Path, optional)`
+
+Directory in which to save model checkpoint and configuration file. If not specified, will save to a folder called 'zamba_{model_name}' in your working directory.
+
+#### `skip_load_validation (bool, optional)`
+
+By default, before kicking off training `zamba` will iterate through all of the videos in the training data and verify that each can be loaded. Setting `skip_load_verification` to `True` skips this step. Validation can be very time intensive depending on the number of videos. It is recommended to run validation once, but not on future training runs if the videos have not changed. Defaults to `False`
+
+#### `from_scratch (bool, optional)`
+
+Whether to instantiate the model with base weights. This means starting from the imagenet weights for image based models and the Kinetics weights for video models. Only used if labels is not None. Defaults to `False`
+
+#### `predict_all_zamba_species (bool, optional)`
+
+Whether the species outputted by the model should be all zamba species. If you want the model classes to only be the species in your labels file, set to `False`. Only used if labels is not `None`. If either `predict_all_zamba_species` is `False` or the labels contain species that are not in the model, the model head will be replaced. Defaults to `True`.
+
+## Specifying advanced configurations in Python
```

Comment:
I think this entire section can be moved/combined with the next page on using yaml configuration files

Reply:
Agreed! I actually think this whole section is mostly repetitive now. I'll just add the full python code for specifying the same defaults as the YAML file for reference at the end of `yaml_config.md`, and get rid of the rest of this section